packages/wallet-core/src/store/actions/updateFees.ts

```diff
@@ -8,7 +7,7 @@ export const updateFees = async (context: ActionContext, { asset }: { asset: Ass
   const network = state.activeNetwork;
   const walletId = state.activeWalletId;
 
-  const chainId = getAsset(network, asset).chain;
+  const chainId = getters.cryptoassets[asset].chain;
```

Comment:
I just think that maybe we can handle the case when we don't have getters.cryptoassets[asset]

Reply:
I agree. However there shouldn't be a case in which an asset is used and it's missing in the store. If that happens, there is a bigger, greater issue that must be fixed.